### PR TITLE
[feat] add both license and contributing information

### PR DESCRIPTION
**Pull request by CBID2**
This addresses the final step of issue #1. See PRs #14 and #13 for context.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,32 +1,18 @@
-This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app).
+# License
```
Comment (EmmaDawsonDev) [inline README.md:1]:
```suggestion
# Accessible For All
```

File: README.md [hunk 2]
```diff
@@ -1,32 +1,18 @@
...
 
-## Getting Started
+## License
 
-First, run the development server:
+AccessibleWebDev is licensed under the MIT License - see [License](https://github.com/AccessibleForAll/AccessibleWebDev/blob/main/LICENSE)
```
Comment (EmmaDawsonDev) [inline README.md:5]:
```suggestion
AccessibleForAll is licensed under the MIT License - see [License](https://github.com/AccessibleForAll/AccessibleForAll/blob/main/LICENSE)
```

Would you be able to go through and update all the links?

**Review by EmmaDawsonDev — CHANGES_REQUESTED** [review]
Thanks Chrissy, this is exactly what I meant :) If you can update all the links to this repo then this will be ready to merge.

**Comment by CBID2** [comment]
> Thanks Chrissy, this is exactly what I meant :) If you can update all the links to this repo then this will be ready to merge.

Will do! :)

File: README.md (hunk 2 above)
Comment (CBID2) [reply README.md:5]: Sure

**Comment by CBID2** [comment]
> > Thanks Chrissy, this is exactly what I meant :) If you can update all the links to this repo then this will be ready to merge.
> 
> Will do! :)

Hi @EmmaDawsonDev! :) I think I go it! :)

**Review by EmmaDawsonDev — APPROVED** [review]
Thanks Chrissy!

**Comment by CBID2** [comment]
> Thanks Chrissy!

You're welcome @EmmaDawsonDev

**Pull request merged**